app/controllers/application_controller.rb

```diff
@@ -1,6 +1,4 @@
 class ApplicationController < ActionController::Base
-  include HttpAuthConcern
```

Comment:
Am I right in saying that this will expose the admin, where we also use the sign-in bypass functionality? Could we include the concern in the base admin controller? I am not clear how that would affect the TID omniauth callbacks controller